Dockerfile

```diff
@@ -38,14 +35,28 @@ RUN export INSTALL_PACKAGES="build-essential curl fontconfig perl python3-dev py
 		zref &&\
 	mktexfmt xelatex.fmt &&\
 	curl -H 'User-Agent: stop checking' -L http://www.sbl-site.org/Fonts/SBL_Hbrw.ttf > /usr/local/share/fonts/SBL_Hbrw.ttf &&\
-	fc-cache &&\
-	cd /usr/src/app &&\
-	python3 setup.py install &&\
-	mkdir data &&\
-	./collectcontexts.py --bhsa /bhsa --module c &&\
-	apt-get remove -qq $INSTALL_PACKAGES &&\
-	apt-get -qq autoremove &&\
-	rm -rf /var/lib/apt/lists/*
+	fc-cache
+
+RUN svn export https://github.com/ETCBC/peshitta/trunk/tf/0.2 /syriac/c
+RUN svn export https://github.com/CenterBLC/LXX/trunk/tf/1935 /greek/c
+
+COPY setup.py README.md /usr/src/app/
+
+RUN cd /usr/src/app && python3 setup.py install
+RUN cd /usr/src/app && mkdir data
+
+COPY hebrewreader.py hebrewreaderserver.py collectcontexts.py minitf.py *.html /usr/src/app/
+RUN cd /usr/src/app && ./collectcontexts.py --bhsa /bhsa --module c --lang hebrew
+RUN cd /usr/src/app && ./collectcontexts.py --bhsa /syriac --module c --lang syriac
+RUN cd /usr/src/app && ./collectcontexts.py --bhsa /greek --module c --lang greek
+
+
+COPY NotoSansSyriac-Regular.ttf /usr/local/share/fonts/syriac_2.ttf
+COPY NotoSerif-Regular.ttf /usr/local/share/fonts/greek.ttf
```

Comment:
A couple of things:

- If possible I would prefer to download the fonts (as is done with the Hebrew one); this makes sure that we always have the latest version.
- Please keep original font names (i.e. `NotoSansSyriac-Regular.ttf` instead of `syriac_2.ttf`).
- Does Noto Serif Regular have all the required glyphs? If I enter the LXX of Gen 1:1 on https://fonts.google.com/noto/specimen/Noto+Serif/tester I see empty boxes for the first letter of the first and third word. Why not use [SBL Greek](https://www.sbl-site.org/educational/biblicalfonts.aspx), as we are using SBL Hebrew already?
- At the bottom of index.html there is a note about the fonts that are being used; please add these.